builds.json

```diff
@@ -2,6 +2,65 @@
   "builds" :
   [
     {
+      "version": "7.0",
+      "packages": [
+        {
+          "package": "org.ga4gh.models",
+          "python_package": "ga4gh",
+          "version": "3.0.0",
+          "dependencies": []
+        },
+        {
+          "package": "org.gel.models.metrics.avro",
+          "python_package": "metrics",
+          "version": "1.1.0",
+          "dependencies": []
+        },
+        {
+          "package": "org.gel.models.participant.avro",
+          "python_package": "participant",
+          "version": "1.1.0",
+          "dependencies": []
+        },
+        {
+          "package": "org.gel.models.report.avro",
+          "python_package": "reports",
+          "version": "6.0.0",
+          "dependencies": [
+            "org.gel.models.participant.avro"
+          ]
+        },
+        {
+          "package": "org.gel.models.system.avro",
+          "python_package": "system",
+          "version": "0.1.0",
+          "dependencies": []
+        },
+        {
+          "package": "org.opencb.biodata.models.variant.avro",
+          "python_package": "opencb",
+          "version": "1.3.0",
+          "dependencies": []
+        },
+        {
+          "package": "org.gel.models.coverage.avro",
+          "python_package": "coverage",
+          "version": "0.1.0",
+          "dependencies": []
+        },
+        {
+          "package": "org.gel.models.cva.avro",
+          "python_package": "cva",
+          "version": "1.0.0",
```

Comment:
This version needs to be increased as we don't want to override the current `cva 1.0.0` working witjh reports 5.0.0. I suggest 1.1.0.